Tests/test_Restriction.py

```diff
@@ -229,27 +229,27 @@ def test_comparisons(self):
         # Comparison of iso- and neoschizomers
         self.assertEqual(Acc65I, Acc65I)
         self.assertNotEqual(Acc65I, KpnI)
-        self.assertFalse(Acc65I == Asp718I)
+        self.assertEqual(Acc65I == Asp718I, False)
         # self.assertNotEqual(Acc65I, Asp718I) it doesn't work as expected
-        self.assertFalse(Acc65I != Asp718I)
```

Comment:
This and the line above look like the problem cases, and I think that neither ``self.assertEqual`` nor ``self.assertNotEqual`` will work here? I've not tried that, but I did look at the source code:

https://github.com/python/cpython/blob/master/Lib/unittest/case.py

However, I think the originals using ``self.assertFalse`` are actually clearer than ``self.assertEqual(..., False)``.

I'm going to ask for a second opinion, but I would be inclined to leave these two lines as they are. We could add the special comment ``# noqa: A500`` to tell flake8 to ignore the warning from flake8-assertive (if installed).

@MarkusPiotrowski could you comment?

Reply:
Thanks for review.
Should I wait for his response?

Reply:
Wait for now - the second opinion might be different.

Reply:
`Bio.Restriction` overwrites the `==` and `!=` comparisons for some special meaning regarding neo- and isoschizomers. So what is actually tested here are the comparison operators and thus I think that the original tests are indeed clearer.

Reply:
Thanks Markus.

@lacrosse91 could you revert those two lines (and optionally add the ``# noqa: A500`` comment - which should make flake8-assertive happy and document this use of ``assertTrue`` is deliberate).